### DISCUSSION TITLE
Possible Error in Error Message. 😅

### DISCUSSION THREAD

**Discussion by Mike-E-angelo** [community/community#25556] [Actions]
<p>Trying out my “frist” action, and I am getting the following error message:</p>
<pre><code>##[error]Top level 'run:' section is required for /home/runner/work/_actions/dragonspark/action-changelog-dawg/master/action.yml
</code></pre>
<p>However, it would seem the required section is <code>runs</code> not <code>run</code> as specified in the message:</p>
<p><a href="https://help.github.com/en/actions/automating-your-workflow-with-github-actions/metadata-syntax-for-github-actions#runs" rel="nofollow noopener">https://help.github.com/en/actions/automating-your-workflow-with-github-actions/metadata-syntax-for-github-actions#runs</a></p>

**Comment by weide-zhou** [comment, marked as answer]
<p>Much Appreciate  your feedback!</p>
<p>I can repo this, I will raise a ticket internally to confirm whether it’s a typo for word ‘run’, thanks for helping us to improve!</p>

**Comment by Mike-E-angelo** [comment]
<p>Happy to help! :+1:</p>
